package.json

```diff
@@ -13,18 +13,22 @@
     "start:services": "cd services/ && docker compose -f docker-compose.yml -f docker-compose.development.yml up -d --build",
     "start:services:appmap": "cd services/ && docker compose -f docker-compose.yml -f docker-compose.development.yml -f docker-compose.development-appmap.yml up -d --build",
     "start:portal": "npm start --prefix interfaces/Portal/ -- --port 8888",
+    "start:portalicious": "npm start --prefix interfaces/Portalicious/ -- --port 8888",
```

Comment:
I _love_ how this name doesn't help with "tab-command-completion" at all... 😑 😶

Reply:
![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExaDQ0d2c1bWlvcTg0dGJ4eXdqMnpldXJjcG5td2QwdDF4aDAxZDJxbCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/e05GB2c86qgOk/giphy.gif)

Reply:
I guess I could simultaneously rename the `portal` scripts to `old-portal`?